csharp/src/Microsoft.ML.OnnxRuntime/OrtIoBinding.shared.cs

```diff
@@ -65,11 +66,17 @@ internal IntPtr Handle
         /// <param name="allocation">native memory allocation</param>
         public void BindInput(string name, Tensors.TensorElementType elementType, long[] shape, OrtMemoryAllocation allocation)
         {
-            using (var ortValue = OrtValue.CreateTensorValueWithData(allocation.Info,
-                                                                    elementType,
-                                                                    shape,
-                                                                    allocation.Pointer, allocation.Size))
-                BindInputOrOutput(name, ortValue.Handle, true);
+            BindOrtAllocation(name, elementType, shape, allocation, true);
+        }
+
+        /// <summary>
+        /// Bind externally allocated memory as input
```

Comment:
nit: Should this description also include a line similar to the other `BindInput()` overload (`OrtMemoryAllocation continues to own the chunk of native memory and should be alive until the end of execution.`) for completeness ?